samples/cpp/continuous_batching_accuracy/continuous_batching_accuracy.cpp

```diff
@@ -67,17 +67,16 @@ int main(int argc, char* argv[]) try {
 
     // Perform the inference
     
-    ov::genai::SchedulerConfig scheduler_config {
-        // batch size
-        .max_num_batched_tokens = 32,
-        // cache params
-        .num_kv_blocks = 364,
-        .block_size = 32,
-        // mode - vLLM or dynamic_split_fuse
-        .dynamic_split_fuse = dynamic_split_fuse,
-        // vLLM specific params
-        .max_num_seqs = 2,
-    };
```

Comment:
I applied your change in [continuous batching CI PR](https://github.com/openvinotoolkit/openvino.genai/pull/572/files#diff-fc02bdbb2db27e61c8c30e17b2e2f406b7bbebea3355917323278bb851d735cbR79) as I need this sample to compile there already.

Reply:
Yeah, sorry...

Reply:
No problem at all, just for you information